goosebit/settings/args.py

```diff
@@ -0,0 +1,5 @@
+import argparse
```

Comment:
This is not where I would expect argument parsing. IMHO this should be in `__main__.py`.